etc/ci_test.sh

```diff
@@ -9,17 +9,18 @@ mkdir -p coverage
 #
 cd pkg/JuliaInterface
 pwd
+# Force recompilation of JuliaInterface with coverage instrumentation
+CFLAGS=--coverage FORCE_JULIAINTERFACE_COMPILATION=true ${GAP} --nointeract
 ${GAP} makedoc.g
 ${GAP} --cover ../../coverage/JuliaInterface.coverage tst/testall.g || AnyFailures=Yes
-gcov -o gen/src/.libs src/*.c*
+gcov -o gen/src/ src/*.c*
 cd ../..
```

Comment:
```suggestion
# Force recompilation of JuliaInterface without coverage instrumentation
FORCE_JULIAINTERFACE_COMPILATION=true ${GAP} --nointeract
cd ../..
```

Maybe we should add something like this? Otherwise, someone running `]test` locally will end up with a JuliaInterface with coverage instrumentation. If this is not desireable, this patch should re-compile a "usual" JuliaInterface

Reply:
Good point. But for that I think doing `make clean` would suffice?

Reply:
yeah it does. changed now